lib/Gocdb_Services/ServiceService.php

```diff
@@ -506,7 +506,6 @@ public function editService(\Service $se, $newValues, \User $user = null) {
         $st = $this->getServiceType ( $newValues ['serviceType'] );
 
         $this->validate ( $newValues ['SE'], 'service' );
-        $this->validateEndpointUrl ( $newValues ['endpointUrl'] );
```

Comment:
Where does `$newValues ['SE'] ['URL']` (formerly `$newValues ['endpointUrl']`) get validated if not here?

Reply:
In the line above. It is now part of the $newValues ['SE'] array